src/js/common/components/form-elements/ErrorableCheckbox.jsx

```diff
@@ -90,7 +90,7 @@ ErrorableCheckbox.propTypes = {
   checked: React.PropTypes.bool,
   errorMessage: React.PropTypes.string,
   name: React.PropTypes.string,
-  label: React.PropTypes.string.isRequired,
+  label: React.PropTypes.object.isRequired,
```

Comment:
Then wouldn't labels that are still strings trigger a warning about the type? This should probably be:

```
label: React.PropTypes.oneOfType([
  React.PropTypes.string,
  React.PropTypes.object
]).isRequired,
```